hazelcast/src/main/java/com/baeldung/hazelcast/cluster/ServerNode.java

```diff
@@ -0,0 +1,23 @@
+package com.baeldung.hazelcast.cluster;
+
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.core.IdGenerator;
+
+public class ServerNode {
+    private static final Logger logger = LoggerFactory.getLogger(ServerNode.class);
+
+    public static void main(String[] args) {
+        HazelcastInstance hazelcastInstance = Hazelcast.newHazelcastInstance();
+        Map<Long, String> countryMap = hazelcastInstance.getMap("country");
+        IdGenerator idGenerator = hazelcastInstance.getIdGenerator("newid");
+        Long countryIdGenerator = idGenerator.newId() == 0L ? 1L : idGenerator.newId();
```

Comment:
Why are we using this check? The IdGenerator is the cluster wide. It will generate unique identifier for each cluster member. Probably you can add 10 entries using 10 unique identifiers with dummy values as:
```
IdGenerator idGenerator = hazelcastInstance.getIdGenerator("newid");
for (int i = 0; i < 10; i++) {
    map.put(idGenerator.newId(), "message" + 1);
}
```

Now you can print all the map data in Hazelcast client. Makes sense?